mage_integrations/mage_integrations/sources/salesforce/client/tap_salesforce/__init__.py

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env python3
+from __future__ import annotations
+
+import asyncio
+import concurrent.futures
+import json
 import sys
-from typing import List
+from copy import deepcopy
 
 import singer
 import singer.utils as singer_utils
 from singer import metadata, metrics
 
-from mage_integrations.sources.messages import write_schema
-from mage_integrations.sources.salesforce.client.tap_salesforce import salesforce
+import mage_integrations.sources.salesforce.client.tap_salesforce.salesforce as salespath
```

Comment:
why do you need to rename it?

Reply:
Removed in 8cc2a0e2931ffc6e60ebd0611b0fa1521525ccda